common/test/acceptance/pages/studio/html_component_editor.py

```diff
@@ -238,6 +238,32 @@ def save_content(self):
         """
         click_css(self, '.action-save')
 
+    def open_image_modal(self):
+        """
+        Clicks and in insert image button
+        """
+        # self.q(css='div i[class="mce-ico mce-i-image"]').click()
+        click_css(self, 'div i[class="mce-ico mce-i-image"]')
+
+    def upload_image(self, file_name):
+        """
+        Upload the image via TinyMCE editor.
+        Upload image and add description and click save to upload image via TinyMCE editor.
+        """
+        file_input_css = "[type='file']"
+
+        # select file input element and change visibility to add file.
+        self.browser.execute_script('$("{}").css("display","block");'.format(file_input_css))
+        self.wait_for_element_visibility(file_input_css, "Input is visible")
```

Comment:
The variable is used here `file_input_css`

Reply:
Sorry. Skipped it.